.github/vacuum-ruleset.yaml

```
@@ -602,3 +602,21 @@ rules:
         then:
             function: typedEnum
         type: validation
+    no-period-in-summary:
+        category:
+            id: operations
```

🔧 Apparently, categories are supposed to have a description as well.
```suggestion
            description: Operations are the core of the contract, they define paths and HTTP methods. These rules check operations have been well constructed, looks for operationId, parameter, schema and return types in depth.
            id: operations
```